lang/python/python-mako/Makefile

```diff
@@ -0,0 +1,53 @@
+#
+# Copyright (C) 2017 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python-mako
+PKG_VERSION:=1.1.3
+PKG_RELEASE:=1
+
+PKG_SOURCE:=Mako-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://files.pythonhosted.org/packages/source/m/mako/
+PKG_HASH:=8195c8c1400ceb53496064314c6736719c6f25e7479cd24c77be3d9361cddc27
+
+HOST_BUILD_DEPENDS:=python3/host
+
+PKG_UNPACK=$(HOST_TAR) -C $(PKG_BUILD_DIR) --strip-components=1 -xzf $(DL_DIR)/$(PKG_SOURCE)
```

Comment:
This is not necessary, right?

Reply:
Removing this line breaks the build, the source archive contains a directory "Mako-1.1.3" while the buildsystems expects the unpacked source in "python-mako-1.1.3".
Please let me know whether there is a better solution for the problem, being new to OpenWRT I am still struggeling to understand the buildsystem despite reading lots of docs, but some package-specific problems still seem hard to solve for me, and again, I copied these lines from https://gist.github.com/lucize/b0691a79beff66a9ed2163e9f3999808#file-makefile-L22

Reply:
i remember that PR for Mako and OpenWrt video;
it was a generally good work-set, but a lot of things needed a bit of polish;
i think the author did not have the time/energy to finish polishing things;

maybe here, it may be a good alternative to do `PKG_BUILD_DIR` instead;
i think we were doing `PKG_UNPACK` here when we had both the python2/python3 variants of packages;
no idea if that was the best approach, but it was what we did at the time;

Reply:
> i remember that PR for Mako and OpenWrt video;
> it was a generally good work-set, but a lot of things needed a bit of polish;
> i think the author did not have the time/energy to finish polishing things;

Being new to Open WRT, but not new to package Kodi ( https://github.com/buildroot/buildroot/commits?author=bkuhls ), I will try to produce a working Kodi package for OpenWRT
(x86_64/gbm/OpenGLES): https://github.com/bkuhls/OpenWrt_video/commits/master